core/shared/src/main/scala/spire/std/sortedMap.scala

```diff
@@ -0,0 +1,141 @@
+package spire
```

Comment:
Making this a bit uglier: the implementation is basically identical as for Map.  Is there a pattern elsewhere in spire that should be followed here?